fastify.js

```diff
@@ -760,10 +760,13 @@ function fastify (options) {
       const request = new Request(id, null, req, null, childLogger, onBadUrlContext)
       const reply = new Reply(res, request, childLogger)
 
+      const customMessage = 'incoming request'
```

Comment:
instantiate a variable which is only needed in the if block?

Reply:
One of my biggest struggles is trying figure out where the customizable message will originate from.  Please let me know if you have any suggestions.  Thank you for your feedback.